flymake-stylelint.el

```diff
@@ -127,8 +123,8 @@ Create linter process for SOURCE-BUFFER which invokes CALLBACK once linter is fi
          :command (if flymake-stylelint-executable-args
                       ;; stylelint will glob `""' which lints the entire directory :-|
                       ;; todo: check out the `--aei' flag
-                      (list flymake-stylelint-executable-name "--no-color" "--stdin-filename" (buffer-name source-buffer) flymake-stylelint-executable-args)
-                    (list flymake-stylelint-executable-name "--no-color" "--stdin-filename" (buffer-name source-buffer)))
+                      (list flymake-stylelint-executable-name "--formatter=unix" "--no-color" "--stdin-filename" (buffer-name source-buffer) flymake-stylelint-executable-args)
```

Comment:
hm, what happens if `flymake-stylelint-executable-args` contains `"--formatter"` already?

Reply:
Good point. If `--formatter` is duplicated, Stylelint will fail with the error message `Error: The flag --formatter can only be set once`.

But it does not seem almost likely to me because the current `flymake-stylelint` only works with `--formatter=string` (default). For example, `(setq flymake-stylelint-executable-args "--formatter=json")` would make nonsense.

Reply:
ok that makes sense!